web_monitoring/html_diff_render.py

```diff
@@ -102,7 +102,8 @@ def html_diff_render(a_text, b_text):
     replacements_new.update(replacements_old)
     soup_new = _add_undiffable_content(soup_new, replacements_new)
 
-    return soup_new.prettify(formatter='minimal')
+    # TODO Count changes.
+    return {'diff': soup_new.prettify(formatter='minimal')}
```

Comment:
I think, in this version, the right thing is `len(soup_new.find_all('ins')) + len(soup_new.find_all('del'))`. We’d probably be fine without it, too.